strictEquals.js

```diff
@@ -0,0 +1,4 @@
+const strictEquals = (valOne, valTwo) => {
+  return (typeof valOne == typeof valTwo && valOne == valTwo)
+}
+module.exports = strictEquals
```

Comment:
Nice! Of course since this is a one line return function you could use fat arrow's implicit return

```JavaScript
const strictEquals = (valOne, valTwo) => typeof valOne == typeof valTwo && valOne == valTwo
```